### PR TITLE
Add scheme-python bridge test

### PR DESCRIPTION
**Pull request by linas**
Extremely simple test to verify that `python-eval` doesn't crash.

### CODE REVIEW

File: tests/scm/CMakeLists.txt [hunk 1]
```diff
@@ -37,3 +37,22 @@ SET_PROPERTY(TEST SCMOpenogTestRunnerFail PROPERTY WILL_FAIL TRUE)
...
+	MESSAGE(STATUS "Project source dir: ${PROJECT_SOURCE_DIR}")
+	EXECUTE_PROCESS(COMMAND guile -c "(display %load-path)" OUTPUT_VARIABLE GLP)
+	MESSAGE(STATUS "Guile load path: ${GLP}")
+	SET(ENV{GUILE_LOAD_PATH} ${PROJECT_SOURCE_DIR}/opencog/scm:${PROJECT_SOURCE_DIR}/opencog/scm/opencog)
+	EXECUTE_PROCESS(COMMAND guile -c "(display %load-path)" OUTPUT_VARIABLE GLF)
+	MESSAGE(STATUS "Full guile load path: ${GLF}")
+	# EXECUTE_PROCESS(COMMAND ls -la ${PROJECT_SOURCE_DIR}/opencog/scm OUTPUT_VARIABLE SDL)
+	# MESSAGE(STATUS "Scheme directory contents: ${SDL}")
+	SET_PROPERTY(TEST SCMOpenogPython
+		PROPERTY ENVIRONMENT "GUILE_LOAD_PATH=${PROJECT_SOURCE_DIR}/opencog/scm:${PROJECT_SOURCE_DIR}/opencog/scm/opencog")
```
Comment (AmeBel) [inline tests/scm/CMakeLists.txt]: PROJECT_SOURCE_DIR/opencog/scm doesn't have the write structure for modules. It should be PROJECT_BINARY_DIR/opencog/scm.

Comment (linas) [reply tests/scm/CMakeLists.txt]: trying now

Comment (linas) [reply tests/scm/CMakeLists.txt]: Thanks @AmeBel but that is still not working

Comment (AmeBel) [reply tests/scm/CMakeLists.txt]: I rebased this pr on the branch of https://github.com/opencog/atomspace/pull/1975 and it is passing

**Comment by linas** [comment]
obsoleted by #1977

**Pull request closed without merging**